### DISCUSSION TITLE
A way to disable many logging?

### DISCUSSION THREAD

**Discussion by zkollz** [LSPosed/LSPosed#1776] [Q&A]
Why isn't there a voluntary option to disable so many module logs and verbose logs?

**Comment by ghost** [comment]
If you installed Debug version yon can't disable verbose log, but if you installed release in Settings / Framework there is Disable verbose log, just enable it
![IMG_20220320_204234](https://user-images.githubusercontent.com/3065939/159179901-a57e76d9-f04c-4529-9861-47c1c14125a8.jpg)
